core/app/modules/kit-library/assets/js/pages/index/index.js

```diff
@@ -70,6 +71,45 @@ function useMenuItems( path ) {
 	}, [ path ] );
 }
 
+/**
+ * Update and read the query param from the url
+ *
+ * @param queryParams
+ * @param setQueryParams
+ * @param exclude
+ */
+function useRouterQueryParams( queryParams, setQueryParams, exclude = [] ) {
+	const location = useLocation();
+
+	useEffect( () => {
+		const filteredQueryParams = Object.fromEntries(
+			Object.entries( queryParams )
+				.filter( ( [ key, item ] ) => ! exclude.includes( key ) && item )
+		);
+
+		history.replaceState(
+			undefined,
+			undefined,
+			decodeURI(
+				`#${ wp.url.addQueryArgs( location.pathname.split( '?' )[ 0 ] || '/', filteredQueryParams ) }`
```

Comment:
Just out of curiosity, do you use the filter query params as a URL hash params instead of real query params?

If so, why not using them as regular params? Any special reason?
Otherwise, why is the hastag there?